rollup.config.js

```diff
@@ -38,7 +38,7 @@ function bundleConfig(name, entryFile) {
       replace({
         preventAssignment: true,
         values: {
-          'process.env.NODE_ENV': process.env.NODE_ENV,
+          'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV),
```

Comment:
This ensures it is still `undefined` if not provided, but it gets quoted as an inline string if it is.